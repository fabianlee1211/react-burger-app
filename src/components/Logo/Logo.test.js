import React from 'react';
import { shallow } from 'enzyme';
import Logo from './Logo';

describe('<Logo />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Logo />);
  });

  it('renders Logo correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('renders a div and image', () => {
    const logo = wrapper.find('div.Logo');
    expect(logo).toHaveLength(1);
    expect(wrapper.find('img')).toHaveLength(1);
  });

  it('receives height props', () => {
    wrapper.setProps({ height: 10 });
    const logo = wrapper.find('div.Logo');
    expect(logo.props().style.height).toBe(10);
  });
});
