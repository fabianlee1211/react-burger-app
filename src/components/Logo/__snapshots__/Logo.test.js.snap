// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders Logo correctly 1`] = `
<div
  className="Logo"
  style={
    Object {
      "height": undefined,
    }
  }
>
  <img
    alt="MyBurger"
    src="burger-logo.png"
  />
</div>
`;
